### PR TITLE
Use imports instead of state.modules

### PR DESCRIPTION
**Pull request by remcohaszing**
We always specify this. It looks a bit cleaner to use regular imports to access these modules.

### CODE REVIEW

File: src/getVariants.ts [hunk 1]
```diff
@@ -1,26 +1,27 @@
...
-  function escape(className: string): string {
-    const node = state.modules.postcssSelectorParser.module.className({ value: className });
-    return node.value;
-  }
+function escape(className: string): string {
+  const node = postcssSelectorParser.className({ value: className });
+  return node.value;
+}
 
+export function getVariants(state: JitState): Record<string, string | null> {
```
Comment (mhsdesign) [inline src/getVariants.ts:15]: what do you think of just passing variantMap and separator to getVariants?
seems even cleaner

Comment (remcohaszing) [reply src/getVariants.ts]: I passed `jitContext` instead, as both `variantMap` and `separator` can be derived from that.

File: src/getVariants.ts [hunk 2]
```diff
@@ -32,7 +32,7 @@ export function getVariants(state: JitState): Record<string, string | null> {
       const container = root.clone();
       const returnValue = fn({
         container,
-        separator: state.separator!,
+        separator: jitContext.tailwindConfig.separator!,
```
Comment (mhsdesign) [inline src/getVariants.ts:35]: smart ^^

**Review by mhsdesign — APPROVED** [review]
Fine by reading!

**Pull request merged**